pkg/cloud/vsphere/context/cluster_context.go

```diff
@@ -102,7 +103,7 @@ func NewClusterContext(params *ClusterContextParams) (*ClusterContext, error) {
 	if logr == nil {
 		logr = klogr.New().WithName("default-logger")
 	}
-	logr = logr.WithName(params.Cluster.APIVersion).WithName(params.Cluster.Namespace).WithName(params.Cluster.Name)
+	logr = logr.WithName(params.Cluster.Name)
```

Comment:
Hi @moshloop,

Per the conversation from #322, please leave this alone for now.